docker/Dockerfile.prod

```
@@ -2,9 +2,14 @@
 FROM node:24-slim AS builder
 WORKDIR /talawa-admin
 
+<<<<<<< HEAD
```

Hadolint (2.14.0)

[error] unexpected '<'
expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input

(DL1000)